config/initializers/autoloading.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# Namespaces have to exist so they can be registered as a custom namespace: https://guides.rubyonrails.org/v8.0/autoloading_and_reloading_constants.html#custom-namespaces
+
+# Build form instances from Cocina objects & DB-persisted data
+module Builders; end
+# Generate Cocina from scalars
+module Generators; end
+# Import hash data (froma JSON export) to DB-persisted models
+module Importers; end
+# Map Cocina objects to form objects
+module Mappers; end
+# Send messages to the expected recipients at the expected time
+module Messengers; end
+# Roundtrip Cocina objects to ensure they can be converted to a form instance and back without lossiness
+module Roundtrippers; end
+# Synchronize Cocina to DB-persisted models
+module Synchronizers; end
```

Comment:
If we want to take any ideas forward at some point, I think we'd probably want to amend the README to include this information and make clear what these entities are in H3.